eclipse-scout-core/src/dataobject/ValueDo.ts

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright (c) 2010, 2023 BSI Business Systems Integration AG
+ *
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+import {DoEntity} from './DoEntity';
+
+/**
+ * <strong>Wrapper data object for a generic value.</strong>
```

Comment:
Please don't create titles using strong, Seperating the first line with an empty line is sufficient.